context.R

```diff
@@ -84,7 +84,7 @@ contextWithRmd <- function(testcases={}, preExec={}) {
                  tryCatch({
                      eval(substitute(preExec), envir = test_env$clean_env)
                      # We don't use source, because otherwise syntax errors leak the location of the student code
-                     test_env$parsed_code <- parse(text = knitr::purl(text = read_lines(student_code)))
+                     test_env$parsed_code <- parse(text = knitr::purl(text = read_lines(student_code), quiet=TRUE))
```

Comment:
Please refrain from including unrelated changes in a pull request. You don't have to do it this time, but try to split up pull requests into logical chunks in the future.

Reply:
Ok I will keep this in mind